docs/api/api_blueprint.apib

```diff
@@ -211,7 +211,7 @@ Create a new user using an email, password and an optional name.
 
 ## User Details [/v1/users/{user_id}]
 + Parameters
-    + user_id: 2 (integer) - ID of the user in the form of an integer
+    + user_id: 1 (integer) - ID of the user in the form of an integer
```

Comment:
Please do not change this, user no. 1 is used to test all the endpoints, deleting it would make the jwt token invalid 😄